src/main/webapp/search-script.js

```diff
@@ -0,0 +1,84 @@
+function countSearchUnits() {
+  const inputElems = document.getElementsByName("search-units");
+  const arr = []
+  for (let i = 0; i < inputElems.length; i++) {       
+    if (inputElems[i].checked){
+      arr.push(inputElems[i].value);
+    }
+  }
+  return arr;
+}
+
+function countAddUnits() {
+  const inputElems = document.getElementsByName("num-units");
+  const arr = []
+  for (let i = 0; i < inputElems.length; i++) {       
+    if (inputElems[i].checked){
+      arr.push(inputElems[i].value);
+    }
+  }
+  return arr;
+}
+
+function showCourses() {
+  const courseResults = document.getElementById("search-results");
+  const courseName = document.getElementById("search-course").value;
+  const profName = document.getElementById("search-prof").value;
+  const termName = document.getElementById("search-term").value;
+  const units = countSearchUnits();
+  const school = getUserSchool();
+  courseResults.innerHTML = "";
+  const url = new URL("/search", window.location.origin);
+  url.searchParams.set("courseName", courseName);
+  url.searchParams.set("profName", profName);
+  url.searchParams.set("units", units);
+  url.searchParams.set("term", termName);
+  url.searchParams.set("schoolName", getUserSchool())
```

Comment:
Why inline this call but not the others? Be consistent in this please. Either way works

Reply:
Oops - just noticed that I set const school above but forgot to actually call the school variable in this line. Changed!